cmd/kubelet/app/options/options.go

```diff
@@ -242,6 +242,9 @@ func (c *kubeletConfiguration) addFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&c.CgroupsPerQOS, "cgroups-per-qos", c.CgroupsPerQOS, "Enable creation of QoS cgroup hierarchy, if true top level QoS and pod cgroups are created. [default: true]")
 	fs.StringVar(&c.CgroupDriver, "cgroup-driver", c.CgroupDriver, "Driver that the kubelet uses to manipulate cgroups on the host.  Possible values: 'cgroupfs', 'systemd'")
 	fs.StringVar(&c.CgroupRoot, "cgroup-root", c.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
+
+	fs.BoolVar(&c.ExtendedIsolation, "extended-isolation", c.ExtendedIsolation, "Enabled extended isolation eventDispatcher. [default: false]")
```

Comment:
This could be more descriptive e.g.:
s/Enabled extended isolation eventDispatcher./Enable extended isolators feature. When is set to true, kubelet will provide API for managing pod level isolation. This option requires `cgroups-per-qos` set to enable./